iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Switch.java

```diff
@@ -53,4 +56,44 @@
      * @param retained the retain status of the switch
      */
     void setRetained(boolean retained);
+
+    default List<OperationalLimits> getOperationalLimits1() {
+        return Collections.emptyList();
+    }
+
+    default <L extends OperationalLimits> L getOperationalLimits1(LimitType limitType, Class<L> limitClazz) {
+        return null;
+    }
+
+    default CurrentLimitsAdder newCurrentLimits1() {
+        throw new UnsupportedOperationException();
+    }
+
+    default ApparentPowerLimitsAdder newApparentPowerLimits1() {
+        throw new UnsupportedOperationException();
+    }
+
+    default VoltageLimitsAdder newVoltageLimits1() {
+        throw new UnsupportedOperationException();
+    }
+
+    default List<OperationalLimits> getOperationalLimits2() {
+        return Collections.emptyList();
+    }
+
+    default <L extends OperationalLimits> L getOperationalLimits2(LimitType limitType, Class<L> limitClazz) {
+        return null;
+    }
+
+    default CurrentLimitsAdder newCurrentLimits2() {
+        throw new UnsupportedOperationException();
+    }
+
+    default ApparentPowerLimitsAdder newApparentPowerLimits2() {
+        throw new UnsupportedOperationException();
+    }
+
+    default VoltageLimitsAdder newVoltageLimits2() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
I would refactor a little bit the code: create a OperationLimitsHolder interface, to avoid to declare these methods everywhere?